src/components/Home.js

```diff
@@ -0,0 +1,134 @@
+
+import React from 'react'
```

Comment:
Same suggestion here:
```suggestion
import React, {useState} from 'react'
```